src/ssl_asn1.c

```diff
@@ -225,6 +231,15 @@ static int wolfssl_i2d_asn1_item(void** item, int type, byte* buf)
             len = wolfSSL_i2d_ASN1_BIT_STRING(
                 *(const WOLFSSL_ASN1_BIT_STRING**)item, buf);
             break;
+        case WOLFSSL_ASN1_INTEGER_ASN1:
+            byte *tmp_buf = buf;
```

Comment:
Older compilers don't allow a declaration unless in a block.
Put braces around this case.

Reply:
Done, thanks!